pkg/language/checks.go

```diff
@@ -390,6 +392,165 @@ func (r *Reader) checkPost(method *concepts.Method) {
 	}
 }
 
+func (r *Reader) checkSearch(method *concepts.Method) {
+	// Get the input and output parameters:
+	var in []*concepts.Parameter
+	var out []*concepts.Parameter
+	for _, parameter := range method.Parameters() {
+		if parameter.In() {
+			in = append(in, parameter)
+		}
+		if parameter.Out() {
+			out = append(out, parameter)
+		}
+	}
+
+	// Get the reference to the and to the version:
```

Comment:
```suggestion
	// Get the reference to the owner and to the version:
```
can also drop this comment, it doesn't add any info.

Reply:
same missing word in `checkList`